src/components/Navbar/navbar.less

```
@@ -40,7 +41,9 @@
   }
 
   .username {
+    font-size: 16px;
```

You can use the DS' `unit` mixin to generate the `em` equivalent of these measurements (which is CFPB's preferred unit for spacing and typography):

```suggestion
    font-size: unit((16px / @base-font-size-px), em);
```